cfme/automate/explorer/domain.py

```diff
@@ -494,7 +494,7 @@ def step(self, *args, **kwargs):
         except CandidateNotFound:
             # Try it with regexp (drop the locked to None)
             # That will force reload from database
-            self.obj.locked = None
+            delattr(self.obj, "locked")
```

Comment:
I don't think there is a reason for use of delattr when we can use `del`
```suggestion
            del self.obj.locked
```
This may help with code analysis in some code editors and re-factoring tools.